### PR TITLE
feat: Updates modal new design changes

### PR DESCRIPTION
**Pull request by albinAppsmith**
## Description

Made new design changes for updates modal.

**Design File:-** https://www.figma.com/file/i80YbQLLs5mXthjIHclPpX/Appsmith-Design-System?node-id=4675%3A23087

Fixes [#16552](https://github.com/appsmithorg/appsmith/issues/16552)

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Tested updates modal in local

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/pages/Applications/ProductUpdatesModal/ReleaseComponent.tsx [hunk 1]
```diff
@@ -12,6 +12,26 @@ const StyledContainer = styled.div`
...
+  align-items: baseline;
+  margin-bottom: 16px;
+`;
+
+const Tag = styled.div`
+  display: flex;
+  align-item: center;
+  justify-content: center;
+  padding: 4px 8px;
+  font-size: 13px;
```
Comment (ankitakinger) [inline app/client/src/pages/Applications/ProductUpdatesModal/ReleaseComponent.tsx:26]: should we use 12px here instead?

File: app/client/src/pages/Applications/ProductUpdatesModal/ReleaseComponent.tsx [hunk 2]
```diff
@@ -166,10 +188,13 @@ function ReleaseComponent({ release }: ReleaseProps) {
 
   return descriptionHtml ? (
     <StyledContainer>
+      <TagContainer>
+        <Tag>{tagName}</Tag>
+        <StyledDate>{moment(publishedAt).format("D MMM YYYY")}</StyledDate>
```
Comment (ankitakinger) [inline app/client/src/pages/Applications/ProductUpdatesModal/ReleaseComponent.tsx:193]: Confirming: Is the format correct?

**Review by ankitakinger — APPROVED** [review]
LGTM

**Comment by shadabbuchh** [comment]
Testing for this PR in `ce` is complete. No issues found there.
However `ee` could not be tested as scripts are failing at docker build. Will test this in `ee` once this PR is merger.

**Pull request merged**